stackstorm/handler.py

```diff
@@ -182,22 +218,35 @@ def base(event, context, passthrough=False):
     else:
         result = output
 
+    # Log the logs generated by the action. We do that so the actual action logs
+    # (action stderr) end up in CloudWatch
+    output = output or {}
+
+    if output.get('stdout', None):
+        LOG.info('Action stdout: %s' % (output['stdout']))
+
+    if output.get('stderr', None):
+        LOG.info('Action stderr and logs: %s' % (output['stderr']))
+
     return {
         'event': event,
         'live_params': live_params,
         'output': output,
         'result': result
     }
 
+
 # for backwards compatibility
 def stackstorm(*args, **kwargs):
     res = base(*args, **kwargs)
     return res['result']
```

Comment:
Question to @enykeev why `result` only here?

Reply:
Previously, it only returned result, now it also returns initial event data, live params and action output alongside the result. I'm not sure this codepath will ever get executed, really, but better safe than sorry.